src/hooks/surveyHooks.ts

```diff
@@ -41,6 +41,7 @@ export const useAnswerSelection = (id?: string, refetch?: () => void) => {
         variables: { decreaseId: previousSubmission, increaseId: selection }
       });
 
+      Alert.alert(texts.survey.submitSuccess.header, texts.survey.submitSuccess.entry);
```

Comment:
can you please add a line break after the new alert to keep the error part more separated

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/23c0005a21c340e4a54e4b49b1adeec8b24017fe